include/eye_assets.hpp

```diff
@@ -0,0 +1,40 @@
+#pragma once
+
+struct EyeAsset {
+    const char* eyeball;
+    const char* iris;
+    const char* pupil;
+    const char* reflex;
+    const char* upperlid;
+};
+
+// EDIT HERE: 目に用いる画像を変えるときや感情種類を増減するときにここを編集
+constexpr EyeAsset normal_eye = {"/normal_kxr_eyeball.jpg", "/normal_kxr_iris.jpg", "/normal_kxr_pupil.png", "/normal_kxr_reflex.png", "/normal_kxr_upperlid.jpg"};
+constexpr EyeAsset blink_eye = {"/normal_kxr_eyeball.jpg", "/normal_kxr_iris.jpg", "/normal_kxr_pupil.png", "/normal_kxr_reflex.png", "/normal_kxr_upperlid.jpg"};
+constexpr EyeAsset surprised_eye = {"/normal_kxr_eyeball.jpg", "/normal_kxr_iris_surprised.jpg", "/normal_kxr_pupil_surprised.png", "/normal_kxr_reflex_surp.png", "/normal_kxr_upperlid.jpg"};
+constexpr EyeAsset sleepy_eye = {"/normal_kxr_eyeball.jpg", "/normal_kxr_iris_sleepy.png", "/normal_kxr_pupil_sleepy.png", "/white.jpg", "/normal_kxr_upperlid.jpg"};
+constexpr EyeAsset angry_eye = {"/normal_kxr_eyeball.jpg", "/normal_kxr_iris.jpg", "/normal_kxr_pupil.png", "/normal_kxr_reflex.png", "/normal_kxr_upperlid_ldown.jpg"};
+constexpr EyeAsset sad_eye = {"/normal_kxr_eyeball.jpg", "/normal_kxr_iris.jpg", "/normal_kxr_pupil.png", "/normal_kxr_reflex.png", "/normal_kxr_upperlid_rdown.jpg"};
+constexpr EyeAsset happy_eye = {"/normal_kxr_eyeball.jpg", "/white.jpg", "/white.png", "/white.png", "/normal_kxr_upperlid_happy.jpg"};
+
+// EDIT HERE: eye_status（ROSトピックのメッセージ）と それぞれのEyeAssetの対応付け
```

Comment:
Basically, you should split definition and variable declaration because if you include the defintion, you will also declare the variable even when you don't want to use it.

Reply:
I recommend the structure like below.

- `eye_assets.hpp`: definition of EyeAsset
- `eye.hpp`: defintion of EyeManager with constructor which has eye_assets_map and upperlid_position_map as arguments.
- `emotion.hpp`: definition of EmotionManager
- `main.cpp`: declaration of eye_assets_map, upperlid_position_map and declaration of Eye instance with those variables.